Security/src/AuthClient/manifest-windows.yml

```diff
@@ -0,0 +1,19 @@
+﻿---
+applications:
+- name: steeltoe-samples-authclient
+  buildpacks:
+  - hwc_buildpack
```

Comment:
Should use the binary buildpack, see [here](https://github.com/SteeltoeOSS/Samples/blob/latest/Discovery/src/FortuneTeller/FortuneTellerWeb/manifest-windows.yml) for reference. Also applies to the other project.